client/src/App.js

```diff
@@ -1,13 +1,16 @@
 import { Route, Routes } from "react-router-dom";
 import MyPlan from "./pages/MyPlan";
 import Landing from "./pages/Landing";
-import Card from "./Components/UI/Card/Card";
+// import Card from "./Components/UI/Card/Card";
+
+// import DateSelector from "./Components/TaskForm/DateSelector";
```

Comment:
I would recommend not leaving commented-out code (or whole files, like `client/src/Components/TaskForm/Modal.js`) in your pull requests

Reply:
Thank you for the feedback, we had a discussion about this before the PR, but going forward we will take your suggested approach. I have removed the commented out files.